rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -101,6 +110,7 @@
       v-model="showBulkEditCleanSelections"
       :title="$t('Clear bulk selection')"
       append-to-body
+      data-test-id="modal-clean-selections"
```

Comment:
There's no preference between using `data-testid` or `data-test-id` but please be consistent, above all were using `data-testid` and the next changes are using the later.